common/common.go

```diff
@@ -101,7 +101,7 @@ const (
 	CdAndExecCmd    = "cd %s && %s"
 	TagImageCmd     = "%s tag %s %s"
 	PushImageCmd    = "%s push %s"
-	BuildClusterCmd = "%s build -f %s -t %s -b %s ."
+	BuildClusterCmd = "%s build -f %s -t %s -c %s -b %s"
```

Comment:
I think we don't need add -c? fllow Docker build command line?

Reply:
if user set context dir ,we need to set it . when build context is not set, we can use  "."  by default.

Reply:
Why not `sealer build -t xxx .` but `sealer build -t xxx -c .`?  We need context but needn't -c flag I think